document/core/util/mathdef.py

```diff
@@ -1,5 +1,5 @@
 from sphinx.ext.mathbase import math
-from sphinx.ext.mathbase import MathDirective
+from sphinx.directives.patches import MathDirective
```

Comment:
Would this happen to make us compatible with newer versions of Sphinx?

Reply:
Yes :) I have https://github.com/ngzhian/spec/tree/upgrade-sphinx prepped and will send it out soon.